dspace-server-webapp/src/test/java/org/dspace/app/rest/csv/CsvExportIT.java

```diff
@@ -53,14 +53,14 @@ public void metadataExportTestWithoutFileParameterSucceeds() throws Exception {
         Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
                                            .withName("Sub Community")
                                            .build();
-        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1")
```

Comment:
Although it's good to test the new scenario, it's not ok that the current setup is no longer tested

Can you restore the original tests, and add these as separate new tests instead?